be-java/src/commonMain/kotlin/lang/temper/be/java/JavaSpecifics.kt

```diff
@@ -141,7 +141,6 @@ abstract class JavaSpecifics(val majorVersion: Int) : RunnerSpecifics {
         @Suppress("UNCHECKED_CAST") // Assertion that allows backend key lookup below.
         dependencies as Dependencies<JavaBackend>
 
-        val runDir: FilePath
```

Comment:
This is part of fixing java run so it depends neither on local install nor on install of temper-core or others from maven central.

While changing things, I realized runDir wasn't really being used. More detail later.